using.rst

```diff
@@ -1726,4 +1728,4 @@ Create Spec and Concept files
 
 .. |install plugin| image:: images/intellij-screenshots/add_plugin.png
 .. |extract concept| image:: images/intellij-screenshots/etc.gif
-.. |ute_parallel| image:: images/visual_studio_screenshots/UTE_parallelicon-small.png
\ No newline at end of file
+.. |ute_parallel| image:: images/visual_studio_screenshots/UTE_parallelicon-small.png
```

Comment:
This could be a line-ending change. Would you mind taking this off your changeset? Thanks.

Reply:
I am not sure whats causing this or how to remove it.  Maybe the new line I added above, but I dont understand why that would cause this to be "deleted" and readded.

Reply:
I did try removing a new line at the bottom of the page, but other than that.  I am not sure.